src/utils.ts

```diff
@@ -326,8 +326,6 @@ export class Utils {
     if (!n.noResize) delete n.noResize;
     if (!n.noMove) delete n.noMove;
     if (!n.locked) delete n.locked;
-    if (n.w === 1 || n.w === n.minW) delete n.w;

```

Comment:
that is not the right fix. data can stil come with fields missing. the freeze part should be fixed instead